[Feature:US18225] Update license to be MIT

BreaGaudioso opened the pull request:
https://rally1.rallydev.com/#/24755394555d/detail/userstory/53778692751

BreaGaudioso commented:
@serv Updated!

serv commented on rhapsody.js:
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2016 Rhapsody International
+// Code released under the MIT license.
+// See README for more detail.
```
Let's give them the explicit URL for the readme.
https://github.com/Rhapsody/rhapsody.js#the-mit-license-mit
This way even if rhapsody.js is copied to someone else's repo, they can still find the original license.

serv commented:
Check out how gulp build works.
https://github.com/Rhapsody/rhapsody.js/blob/master/gulpfile.js#L12

I think that when you run gulp build, it should also edit the minified files and other source files.

This will automatically add the license comment in other js files.

BreaGaudioso commented:
* Added link to license section of README in comment
* Added the comment to the source file 
* Ran gulp build which added comment in areas specified in the task.

serv commented:
Looks good. Merging.
